tests/js/sentry-test/fixtures/releaseV2.js

```diff
@@ -0,0 +1,84 @@
+export function ReleaseV2(params, healthParams) {
```

Comment:
Does ReleaseV2 have it's own endpoints?

Reply:
On the index page, we are currently using the old one with few additional get params.